Clarify FS GCP capabilities (#91)
doitintl/help: website/docs/flexsave/gcp.mdx

## Capabilities

### Supported commitment types

Flexsave supports:

- all GCP [regions](https://cloud.google.com/compute/docs/regions-zones)

> - most of the [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs

Author: the not supported types are in the following section

> - Most of the [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs

Reviewer: ```suggestion
- Most [machine types](https://cloud.google.com/compute/docs/machine-types) supported by resource-based CUDs (see below for exceptions)
```

What about something like this? I still feel that we should qualify the "most".

Author: as they are immediately adjacent, I think it's fine
